one-shot-device/postman collection/authorize-device-to-channel.postman_collection.json

```diff
@@ -0,0 +1,228 @@
+{
+	"info": {
+		"_postman_id": "20b84a1d-9117-4593-8e01-b97a7690a12a",
+		"name": "is-one-shot-device-registration",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "authentication",
+			"item": [
+				{
+					"name": "prove manager's id",
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{is_url}}/authentication/prove-ownership/{{manager_id}}?api-key={{api-key}}",
+							"host": ["{{is_url}}"],
+							"path": ["authentication", "prove-ownership", "{{manager_id}}"],
+							"query": [
+								{
+									"key": "api-key",
+									"value": "{{api-key}}"
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "authenticate via signed nonce",
+					"protocolProfileBehavior": {
+						"disabledSystemHeaders": {
+							"content-type": true
+						}
+					},
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"signedNonce\":\"8a7c73113028abbc6dc385c0bdc66d477c18b0a47b28a0d9f6c62b1e019e501788d54fa34ba177ebae1ced723465e318f247cef1e09585aa9ca194b9cfec1d0a\"\n}"
+						},
+						"url": {
+							"raw": "{{is_url}}/authentication/prove-ownership/{{manager_id}}?api-key={{api-key}}",
+							"host": ["{{is_url}}"],
+							"path": ["authentication", "prove-ownership", "{{manager_id}}"],
+							"query": [
+								{
+									"key": "api-key",
+									"value": "{{api-key}}"
+								}
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Channels",
+			"item": [
+				{
+					"name": "Create channel as manager",
+					"protocolProfileBehavior": {
+						"disabledSystemHeaders": {
+							"content-type": true
+						}
+					},
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							},
+							{
+								"key": "Authorization",
+								"value": "{{manager_jwt}}",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"topics\": [{\"type\":\"smart-audit-log\",\"source\":\"devices in construction sector\"}],\n    \"hasPresharedKey\":true\n}"
+						},
+						"url": {
+							"raw": "{{is_url}}/channels/create?api-key={{api-key}}",
+							"host": ["{{is_url}}"],
+							"path": ["channels", "create"],
+							"query": [
+								{
+									"key": "api-key",
+									"value": "{{api-key}}"
+								}
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Identities",
+			"item": [
+				{
+					"name": "create managers id",
```

Comment:
please add some body of a person as manager

Reply:
having a firstname lastname and some username to identify it. username can be: `one-shot-manager-1`